### ISSUE TITLE
DirectoryNotFoundException:Could not find a part of path '.\.\StreamingAssets\aa' In Unity2021

### ISSUE THREAD

**Issue by chasing2moro** [jelte/be.khepri.play.assetdelivery.addressables#52]
Using BuildPlayerContext.AddAdditionalPathToStreamingAssets in Unity2021. It will not copy aa From Library to StreamingAssets. So it raise a bug :DirectoryNotFoundException:Could not find a part of path '.\.\StreamingAssets\aa'
![lQLPJxZ8N5FS7X7NAorNB-qwDxKlvkPFDrACzJVipICiAA_2026_650](https://user-images.githubusercontent.com/5895421/178634916-fd0a08e0-9eed-4a4c-92f1-36c06a0a585c.png)

**Comment by chasing2moro** [comment]
![lQLPJxZ8N5FS7X7NAorNB-qwDxKlvkPFDrACzJVipICiAA_2026_650](https://user-images.githubusercontent.com/5895421/178634854-109dfa53-d939-41cf-a5a1-662d2179e52d.png)

**Comment by jelte** [comment]
looking at the addressables changelog this is likely due to:
```
Optimised StreamingAssets usage to no longer need to be copied into the project (2021.2+).
```
I'll have a look when I get some time.

**Comment by chasing2moro** [comment]
Forget it.Thank you very much.
I have a backup solution(https://github.com/Unity-Technologies/Addressables-Sample#advancedplay-asset-delivery) to build aab

**Comment by jelte** [comment]
This should be resolved in 1.0.1
